### DISCUSSION TITLE
ast node to string

### DISCUSSION THREAD

**Discussion by neilmi** [antonmedv/expr#333] [Q&A]
hey :)
for my use case i need to convert ast node back to string 
is there any built in code to do that? or only custom code with visitor etc?
i saw dump but i need binary node to be outputted as "x + y" for instance 

thanks :)

**Comment by antonmedv** [comment]
Right now it’s not added. Will be cool to add it.

**Comment by neilmi** [comment]
another question, is there any way to turn off + on strings?

**Reply by antonmedv** [reply]
You can use operator overloading for `func (string, string) string` and return an error in this function instead.

**Reply by neilmi** [reply]
nice,aight, thank u!!
